popup.html

```diff
@@ -15,13 +15,16 @@
     </head>
 
     <body>
+        <div class="container">
         <div class="title-block">
             <h1 style="margin-bottom: .2em;">No More Domains</h1>
             <p style="margin-bottom: 1.75em; margin-top: .2em;">Stop registering new domains.</p>
+            <div class="power-switch">
+                <button onclick="onOff()">Toggle NMD</button>
+            </div>
         </div>
-        <div class="power-switch">
-            <button onclick="onOff()">Toggle NMD</button>
-        </div>
+        
+    </div>
     </body>
 
 </html>
```

Comment:
The button should not be within the `.title-block` div.

Reply:
No it is not in  .title-block 
It is in .container

Reply:
Any other changes ?

Reply:
![image](https://user-images.githubusercontent.com/80921257/193420621-f2c4ffd2-be4f-4e96-89a7-a3ab0e373b5e.png)